### ISSUE TITLE
docker secret support

### ISSUE THREAD

**Issue by eduarddrenth** [eXist-db/exist#2956]
Would it be a good idea to be able to provide admin password via docker secret?

See https://docs.docker.com/engine/swarm/secrets/

**Comment by duncdrum** [comment]
Yes, this is  possible for dockerfiles (without swarmm) using the experimental features of the docker build command. see also https://docs.docker.com/v17.12/engine/reference/commandline/secret/ .

However, this is for the users to implement on their end, our images should ship with the same defaults as on other release channels.

**Comment by duncdrum** [comment]
#2957

**Comment by eduarddrenth** [comment]
Sorry I don't understand your answer. How exactly do I then provide the admin password via docker secret? Then again my question was realy short....

I do know about and use docker secret, for payara and postgres images. In both cases the application has an option to read passwords from /run/secrets/xxx.

As far as I know exist-db does not have such an option. So I can create a secret, but cannot use it.
